.github/workflows/docker.yaml

```diff
@@ -70,7 +70,7 @@ on:
 jobs:
   check:
     runs-on: ubuntu-latest
-    if: ${{ github.actor != 'dependabot[bot]' || contains(github.event.pull_request.labels.*.name, 'docker') || github.event_name == 'push' }}  # ignore the pull request which comes from user depbot.
+    if: ${{ github.actor != 'dependabot[bot]' && (github.event_name != 'pull_request' || !github.event.pull_request.draft ) }}
```

Comment:
I think everyone needs the docker image to test on DEMO before putting the PR on review

Reply:
At least let's keep a label to build the image. If we need to put it in ready for review just to test, we lose the best way to understand what can be reviewed and what cannot